src/components/POLine/Item/ItemView.js

```diff
@@ -27,8 +28,13 @@ class ItemView extends Component {
           <Col xs={6}>
             <KeyValue
               label={<FormattedMessage id="ui-orders.itemDetails.title" />}
-              value={get(poLineDetails, 'title')}
-            />
+            >
+              {get(poLineDetails, 'instanceId') ? (
```

Comment:
too many similar `gets`... I think you can extract to const in `render` like `const instanceId = get(poLineDetails, 'instanceId');` or something like that

Reply:
done